tda/build.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
+deploy_id=$1
+
 set -e 
-virtualenv --python=`which python3.8` env
-source env/bin/activate 
-pip install -r requirements.txt
\ No newline at end of file
+virtualenv --python=`which python3.8` "env$deploy_id"
```

Comment:
do you have an old version of `virtualenv`?  you should be able to do `--python-python3.8` and as long as you've got a modern-ish virtualenv it should be able to replace itself

Reply:
@asottile-sentry  That would be nice but seems like it's already latest
<img width="516" alt="Screenshot 2023-10-21 at 2 19 40 PM" src="https://github.com/sentry-demos/empower/assets/490201/7166f564-a990-4891-a919-db693bf1f03e">

Reply:
that version is ancient!